### PR TITLE
Show last sync date Fixes #1229

### LINKED ISSUE #1229: show last time synced on login screen and setting
show last time synced on login screen and setting so users on a glance can see when . last the tablet data got synced with server... this is to encourage frequent tablet synchronisation with server

### PR DESCRIPTION
**Pull request by ChiragAwale**
Fixes #1229

### CODE REVIEW

File: app/src/main/res/layout/activity_child_login.xml [hunk 1]
```diff
@@ -90,12 +90,20 @@
...
+
+                <TextView
+                    android:id="@+id/tv_lastDateSynced"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:hint="Last Sync Date: 10/10/19"
+                    android:textAlignment="center"
+                    android:textColor="@android:color/white"
+                    android:textSize="10sp" />
             </LinearLayout>
```
Comment (rrijal53) [inline app/src/main/res/layout/activity_child_login.xml]: @ChiragAwale add this text view in other activity_login files too..
Screen Shot 2019-10-14 at 07 18 05

Comment (ChiragAwale) [reply app/src/main/res/layout/activity_child_login.xml]: @rrijal53 gotcha

**Comment by ChiragAwale** [comment]
@leomaxi Gotcha. so once the other pr is merged I will remove this one for the login activity?

**Comment by ChiragAwale** [comment]
@leomaxi Removed the date from login activity. I changed the settings one to the format matching that of login activity. Please review.

**Comment by rrijal53** [comment]
@ChiragAwale Date is still there in child mode.
Screenshot_1571154238

**Comment by ChiragAwale** [comment]
@rrijal53 Sorry about that. Done.

**Pull request merged**